src/java.base/share/classes/sun/security/util/Debug.java

```diff
@@ -208,16 +269,53 @@ public void println(Object obj, String message)
 
     public void println()
     {
-        System.err.println(prefix + ":");
```

Comment:
While we are her we could switch to Platfom logger, that also removes the need for timestamps (and thread info)

Reply:
I'm thinking along the same lines. the` javax.net.debug` property (TLS) already allows the use of `System Logger`. I'd be interested to hear from anyone who uses that Logger, by passing no args to `javax.net.debug`.  No reason why this (java.security.debug) code couldn't be updated also. I'd like to do this in a follow on (hoping to backport this patch to LTS update releases)

I think the Logger experience could do with tweaking also - something I spoke briefly about at the recent OpenJDK Committers' Workshop in Brussels. One option might be a mechanism where the Logger `Level` can be dialed up and down remotely (e.g. jconsole). I've been looking at this with the  `PlatformLoggingMXBean.setLoggerLevel `operation.